Enemyattacks

Dominic-Roser opened the pull request:
the enemy can now use all powerups 

go into level 5 (only functional here currently)

check out **enemypathlvl5.cs**

scroll to **line 84 and uncomment the powerup** you want to test (I recommend one at a time for now)

Sean0z reviewed (commented):
- When you get hit by the water gun, you sometimes significantly speed up. Not sure if this is a bug or not, physics seem a little wonky
- There are a lot of issues when trying to use power ups against the enemy in level 5. Not sure if its intended to work yet though

Dominic-Roser commented:
fixed the strange stretching on rotation, dont worry about the powerups not doing anything rn in lvl5, they all still work, eventually we'll put enemypath back into that player and they'll work again.

also note I added a lil visual response for the rowing bar in lvl5 it looks yucky rn, but we can put in nice art later to make it look cool
